pyerrors/correlators.py

```diff
@@ -42,7 +45,7 @@ class Corr:
 
     __slots__ = ["content", "N", "T", "tag", "prange"]
 
-    def __init__(self, data_input, padding=[0, 0], prange=None):
+    def __init__(self, data_input: Any, padding: list[int]=[0, 0], prange: Optional[list[int]]=None):
```

Comment:
SHouldn't this be `list[Obs]`? Alternatively, this could be something Obs-like, I suppose, such as CObs?